adafruit_turtle.py

```diff
@@ -94,8 +98,11 @@ class Vec2D(tuple):
     #     k*a and a*k multiplication with scalar
     #     |a| absolute value of a
     #     a.rotate(angle) rotation
-    def __init__(self, x, y):
-        super().__init__((x, y))
+    def __new__(cls, x, y):
+        return (x, y)
```

Comment:
I get this error using this version on a PyPortal running the simpletest script in this repo:
```
Adafruit CircuitPython 8.0.0-beta.0 on 2022-08-18; Adafruit PyPortal with samd51j20
Board ID:pyportal
```

```
Traceback (most recent call last):
  File "code.py", line 32, in <module>
TypeError: unsupported types for __sub__: 'tuple', 'tuple'
```
The simpletest is trying to subtract positions here: https://github.com/adafruit/Adafruit_CircuitPython_turtle/blob/9f23b464df772da1565c09a06e2da84ff9da1b53/examples/turtle_simpletest.py#L23

Which turns out not to work because tuple doesn't have support for minus operator. This was odd to me at first because `__sub__` is defined for Vec2D but it turns out because of the change in `__new__`  that these were actually being tuple objects instead of Vec2D objects.

I ended up trying this version with CPython using PyGameDisplay and got the same error from the simpletest where it crashes due to trying to subtract the Vec2D's which are actually tuples instead.

Some of the functionality does work properly though because you can see it draw the first line of the star in the simpletest before crashing. It seems only if you try to utilize some of the other Vec2D functions it will have trouble.